homeassistant/components/netgear/device_tracker.py

```diff
@@ -46,16 +47,19 @@ def new_device_callback() -> None:
     new_device_callback()
 
 
-class NetgearScannerEntity(NetgearBaseEntity, ScannerEntity):
+class NetgearScannerEntity(NetgearDeviceEntity, ScannerEntity):
     """Representation of a device connected to a Netgear router."""
 
+    _attr_has_entity_name = False
+
```

Comment:
```suggestion
```
Is already declared in NetgearDeviceEntity

Reply:
No there it is set to `_attr_has_entity_name = True`, here I need to overwrite that agian to set it to False.
Because it is a tracker entity this is needed.